tests/images.sh

```diff
@@ -2,8 +2,9 @@
 
 set -euo pipefail
 
-export PATH="$(realpath $(dirname $0)/../target/${PROFILE:-debug}):$PATH"
-fixturesdir="$(realpath $(dirname $0)/../fixtures)"
+export PATH
+PATH="$(realpath "$(dirname "$0")/../target/${PROFILE:-debug}"):$PATH"
```

Comment:
It looks like this line is failing in CI and I don't understand why.

Reply:
I found at least [one other CI](https://jenkins-coreos-ci.apps.ocp.fedoraproject.org/blue/rest/organizations/jenkins/pipelines/coreos-installer/branches/PR-1028/runs/4/nodes/223/steps/226/log/?start=0) run for a different PR where this failed. But I think it was ignored exactly because of what ShellCheck wants us to fix here (splitting `export` from setting).

I think the issue is that `target/` isn't present because since https://github.com/coreos/coreos-installer/pull/506, we build in a separate pod from where we build FCOS. We don't actually need to do this though for the CoreOS CI case because `coreos-installer` is [now installed](https://github.com/coreos/coreos-installer/blob/2c08dd141a4dfbc3306733bc6b7845fcc439d73a/.cci.jenkinsfile#L17-L18) in the cosa pod.

It's useful for local development though. So one solution is to change these lines to be conditional on `target/` existing and otherwise assume we want to test the installed binary. Or alternatively if we want to be more strict, add a knob (e.g. env var) to explicitly request testing installed binaries and use it from `.cci.jenkinsfile`.

Reply:
Ohh, good catch.  Yeah, the env var approach seems best for preventing surprises, along with a message saying that we're testing the installed copy.

Reply:
@travier, looks like this CI failure wasn't addressed in your last push.  Is this PR still on your plate?

Reply:
I did not address it indeed

Reply:
Will try to get to it soon

Reply:
Should be fixed now

Reply:
Merging as-is, and will address this in a follow-up.

Reply:
Followup in https://github.com/coreos/coreos-installer/pull/1125.